src/hackerspace_online/tests/test_forms.py

```diff
@@ -598,7 +598,7 @@ def test_signup_via_post_google_signin_change_email_and_revert_back_to_google_em
         self.assertTrue(user.socialaccount_set.exists())
 
         # Change email and then verify
-        form_data = generate_form_data(model_form=ProfileForm, grad_year=timezone.now().date().year + 2)
+        form_data = generate_form_data(model_form=ProfileForm, custom_profile_field=timezone.now().date().year + 2)
```

Comment:
Why is a year still a default?  I think the field can be removed since it's now optional.

Reply:
done